README.md

```diff
@@ -160,6 +160,23 @@ const menuTemplate = new MenuTemplate<MyContext>((ctx) => {
 });
 ```
 
+### Can I use entities or `grammyjs/parse-mode` in the message body?
+
+Yes, you can pass the option `entities`.
+
+This will be resolved automatically to`caption_entities` if you send a `media` message.
```

Comment:
I think I would remove this statement. It's meant to be transparently handled and is the same for `text` and `caption` so I don't think there is much confusion about it.
(But I know the library well, so maybe it's helpful for others?)